api/workAllocation/routes.ts

```diff
@@ -1,7 +1,7 @@
 import { Router } from 'express'
 
 import { getAllCaseWorkers, getAllCaseWorkersForLocation, getCaseWorkersForLocationAndService,
-        getCaseWorkersForService, getTask, postTaskAction, searchCaseWorker, searchTask } from '.'
+        getCaseWorkersForService, getTask, postTaskAction, postTaskSearchForCompletable, searchCaseWorker, searchTask } from '.'
```

Comment:
This `import` set seems to stretch over two lines but not in a standard manner. If they don't fit on a single line, they should all be listed one per line.

Reply:
Ok, moved it to different lines now.